pkg/tests/tasks/security/authorization/common.go

```diff
@@ -0,0 +1,58 @@
+package authorization
+
+import (
+	"fmt"
+
+	"github.com/maistra/maistra-test-tool/pkg/util/check/assert"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/pod"
+	"github.com/maistra/maistra-test-tool/pkg/util/retry"
+	"github.com/maistra/maistra-test-tool/pkg/util/test"
+)
+
+func httpbinRequest(method string, path string, headers ...string) string {
+	headerArgs := ""
+	for _, header := range headers {
+		headerArgs += fmt.Sprintf(` -H "%s"`, header)
+	}
+	return fmt.Sprintf(`curl "http://httpbin:8000%s" -X %s%s -sS -o /dev/null -w "%%%%{http_code}\n"`, path, method, headerArgs)
+}
+
+func verfiyHttpbinRequest(t test.TestHelper, ns string, curlCommand string) {
+	retry.UntilSuccess(t, func(t test.TestHelper) {
+		oc.Exec(t,
+			pod.MatchingSelector("app=sleep", ns),
+			"sleep",
+			curlCommand,
+			assert.OutputContains(
+				"200",
+				"Got expected 200 OK from httpbin",
+				"Expected 200 OK from httpbin, but got a different HTTP code"))
+	})
+}
```

Comment:
This function seems to be a duplicate of `assertRequestAccepted`, with just a slightly different success/failure message. Can't you consolidate the two functions into one?

If you can't consolidate them, then this function needs to be in `ext_auth_test.go`, since it's only used there.

Reply:
Actually, we are using this code in the `deny_test.go`. I will the deny file in the next commit https://github.com/maistra/maistra-test-tool/blob/7840f4a1c93a7473ce673b53ee7e867c2b4ba6a9/pkg/tests/tasks/security/authorization/deny_test.go#L44